packages/@lwc/perf-benchmarks/scripts/build.js

```diff
@@ -114,10 +114,10 @@ function createTachometerJson(htmlFilename, benchmarkName, directoryHash, cpuThr
                                             // `@lwc/perf-benchmarks-components` itself.
                                             'rm -fr ./packages/@lwc/perf-benchmarks-components/{src,scripts}',
                                             `cp -R ${benchmarkComponentsDir}/{src,scripts} ./packages/@lwc/perf-benchmarks-components`,
-                                            'yarn --immutable',
+                                            'if [ -f pnpm-lock.yaml ]; then pnpm install --frozen-lockfile; else yarn --frozen-lockfile; fi',
```

Comment:
```suggestion
                                            'if [ -f pnpm-lock.yaml ]; then pnpm install --frozen-lockfile; else yarn --immutable; fi',
```

Reply:
Seems `--immutable` was used by mistake originally. It's yarn v2. [Yarn classic doesn't mention it.](https://classic.yarnpkg.com/en/docs/cli/install)

Reply:
Yep we messed up. Only yarn v2+ uses `--immutable`.